bots/notify/src/main/java/org/openjdk/skara/bots/notify/IssueUpdater.java

```diff
@@ -59,29 +68,48 @@ public void handleCommits(HostedRepository repository, List<Commit> commits, Bra
                 issue.get().addComment(commitNotification);
                 issue.get().setState(Issue.State.RESOLVED);
 
-                var linkBuilder = Link.create(repository.webUrl(commit.hash()), "Commit")
-                                      .summary(repository.name() + "/" + commit.hash().abbreviate());
-                if (commitIcon != null) {
-                    linkBuilder.iconTitle("Commit");
-                    linkBuilder.iconUrl(commitIcon);
+                if (commitLink) {
+                    var linkBuilder = Link.create(repository.webUrl(commit.hash()), "Commit")
+                                          .summary(repository.name() + "/" + commit.hash().abbreviate());
+                    if (commitIcon != null) {
+                        linkBuilder.iconTitle("Commit");
+                        linkBuilder.iconUrl(commitIcon);
+                    }
+                    issue.get().addLink(linkBuilder.build());
+                }
+
+                if (fixVersion != null) {
```

Comment:
How about turning this around a bit into:

```java
if (fixVersion == null) {
   // Try to find fixVersion from .jceck/conf
}
if (fixVersion != null) {
    issue.get().addFixVersion(fixVersion)
}
```
If you don't sent the `fixVersion` via the bot configuration, then it will be `null`, and then we will read it from `.jcheck/conf`. This avoids the need for the special `<repo>` version.

Reply:
Right, but if you create a fork where you don't want to change the .jcheck/conf file (for merge reasons) you may perhaps not want the fixVersion updated?

Reply:
Good point! Should we then perhaps represent this using two parameters, `boolean shouldUpdateFixVersion` and `String fixVersion`?